### PR TITLE
postgres and version bumps

### PR DESCRIPTION
**Pull request by wstrange**
Updated postgres helm chart to more current 10.5 release
Updated postgres chart for improved storage class handling
Update AM docker image to M6
Update OpenIDM image to use a snapshot until a new Milestone is produced

### CODE REVIEW

File: helm/postgres-openidm/values.yaml [hunk 1]
```diff
@@ -23,17 +19,19 @@ imageTag: "9.5.4"
...
 
-## Persist data to a persitent volume.
+## Persist data
 persistence:
   enabled: true
-  storageClass: generic
   accessMode: ReadWriteOnce
-  size: 8Gi
+
+storageClassName: standard
```
Comment (wajihahmed) [inline helm/postgres-openidm/values.yaml]: Is this supposed to be at top level?

Comment (wstrange) [reply helm/postgres-openidm/values.yaml]: yes -I  moved to top level. There really isn't a good reason to nest this

Comment (wajihahmed) [reply helm/postgres-openidm/values.yaml]: Ok but there is the persistence section and this would be a logic fit under it as before?

Comment (wstrange) [reply helm/postgres-openidm/values.yaml]: I will top level those as well...

Comment (wstrange) [reply helm/postgres-openidm/values.yaml]: done

**Pull request merged**